display/src/main/scala/uk/ac/wellcome/platform/archive/display/files/DisplayStandaloneFile.scala

```diff
@@ -0,0 +1,57 @@
+package uk.ac.wellcome.platform.archive.display.files
+
+import io.circe.generic.extras.JsonKey
+import uk.ac.wellcome.platform.archive.common.storage.models.{StorageManifest, StorageManifestFile}
+
+// We index individual files into their own index in Elasticsearch.
+//
+// For example, somebody might want to query for all the JP2 files created in
+// the "digitised" space in a given month.
+//
+// We don't want to ingest just the DisplayFile model used in the bags API,
+// because that doesn't have any context about the space of external identifier
+// of this particular bag.
+
+case class DisplayAssociatedBag(
```

Comment:
Is it called an `AssociatedBag`  as it's a link to the bag that contains the file?

Reply:
Pretty much, yeah. Alternative names welcome!

Reply:
Yea, we've got the same thing on the Merger, using images as an example.

We use `MergedImage` and `UnmergedImage`, which maybe isn't clearly described.

Thinking about it in that way (the thing, and the link to the thing), could you just have `Bag` and BagLink` or maybe `BagRel`?

Reply:
`Bag` is already used in a bunch of other places – I wonder if `ParentBag` would work? Because there is a sort of child–parent relationship between file and bag.

Reply:
Why not just link? Sorry that comment wasn't very clear, I was suggesting that `Bag` is the [current `Bag`](https://github.com/wellcomecollection/storage-service/blob/master/common/src/main/scala/uk/ac/wellcome/platform/archive/common/bagit/models/Bag.scala) and this is a `Link` to that `Bag`.

So maybe `BagLink` implying relationship, but not implying the type (hierarchical).